sky/serve/autoscalers.py

```diff
@@ -222,17 +223,15 @@ def update_version(self, version: int, spec: 'service_spec.SkyServiceSpec',
         self.downscale_counter = 0
 
     def collect_request_information(
-            self, request_aggregator_info: Dict[str, Any]) -> None:
-        """Collect request information from aggregator for autoscaling.
-
-        request_aggregator_info should be a dict with the following format:
+            self, request_aggregator_info: schemas.RequestsAggregator) -> None:
+        """Collect request information from aggregator for autoscaling."""
+        if isinstance(request_aggregator_info, schemas.TimestampAggregator):
+            self.request_timestamps.extend(request_aggregator_info.timestamps)
+        else:
+            logger.error(f'Unsupported request aggregator type: '
+                         f'{type(request_aggregator_info)}')
+            return
```

Comment:
```suggestion
        if not isinstance(request_aggregator_info, schemas.TimestampAggregator):
            logger.error(f'Unsupported request aggregator type: '
                         f'{type(request_aggregator_info)}')
            return
        self.request_timestamps.extend(request_aggregator_info.timestamps)
```
the following logic in this function also only works if the aggregator is a timestamp aggregator?